packages/cli/src/cli/commands/init.ts

```diff
@@ -93,12 +100,11 @@ const init = async ({
     repo: "pantheon-content-cloud-sdk",
   });
   writeFileSync(path.join(TEMP_DIR_NAME, TAR_FILE_NAME), Buffer.from(data));
-  await sh("tar", [
-    "xvpf",
-    path.join(TEMP_DIR_NAME, TAR_FILE_NAME),
-    "-C",
-    TEMP_DIR_NAME,
-  ]);
+  await sh(
+    "tar",
+    ["xvpf", path.join(TEMP_DIR_NAME, TAR_FILE_NAME), "-C", TEMP_DIR_NAME],
+    !silentLogs,
```

Comment:
May we remove it, since there will be too many logs for extract.

Reply:
I'll add a verbose flag